Source/Core/DolphinLib.props

```diff
@@ -253,6 +253,7 @@
     <ClInclude Include="Core\HW\DVD\DVDMath.h" />
     <ClInclude Include="Core\HW\DVD\DVDThread.h" />
     <ClInclude Include="Core\HW\DVD\FileMonitor.h" />
+	<ClInclude Include="Core\HW\EXI\BBA\BuiltIn.h" />
```

Comment:
plz fix the whitespace in this file (it at least looks screwy on github)

Reply:
Tab vs 4 space... didnt notice. sorry